gnmi/gnmi.go

```diff
@@ -20,23 +20,15 @@ import (
 
 	"github.com/openconfig/lemming/gnmi/gnmit"
 	"github.com/openconfig/lemming/gnmi/oc"
-
-	gnmipb "github.com/openconfig/gnmi/proto/gnmi"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
 )
 
 // Server is a fake gNMI implementation.
 type Server struct {
-	*gnmit.GNMIServer
-	s            *grpc.Server
-	Responses    [][]*gnmipb.SubscribeResponse
-	GetResponses []interface{}
-	Errs         []error
+	*gnmit.GNMIServer[*oc.Root]
 }
 
-func createGNMIServer(targetName string) (*gnmit.GNMIServer, error) {
+func New(srv *grpc.Server, targetName string) (*Server, error) {
```

Comment:
probably just delete this and move it lemming

Reply:
I kept this to keep uniformity with the other protocols